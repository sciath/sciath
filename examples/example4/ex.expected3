@ -- 
@ -- Output from fake application code 
@ -- comment lines 
@ -- 
kspits 42
kspits 58
kspits 21
kspits 33
norm 9.33E+07
