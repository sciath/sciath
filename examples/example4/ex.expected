@ -- 
@ -- Output from fake application code 
@ -- comment lines 
@ -- 
kspits 42
kspits 58
kspits 21
kspits 33
norm 2.33E+05
